source/fuzz/protobufs/spvtoolsfuzz.proto

```diff
@@ -1011,12 +1010,8 @@ message TransformationPermuteFunctionParameters {
   // Function, whose parameters will be permuted
   uint32 function_id = 1;
 
-  // |new_type_id| is a result id of a valid OpTypeFunction instruction.
-  // New type is valid if:
-  //   - it has the same number of operands as the old one
-  //   - function's result type is the same as the old one
-  //   - function's arguments are permuted according to |permutation| vector
-  uint32 new_type_id = 2;
+  // Fresh id for a new type of the function.
```

Comment:
I presume this might not be used if the new function type already exists - worth commenting on.

Reply:
Fixed in 93b8e3a.